bot/resources/tags/venv.md

```diff
@@ -0,0 +1,18 @@
+**Virtual Environments**
+
+Virtual environments are isolated Python environments, which make it easier to keep your system clean and manage dependencies. By default, when activated, only libraries and scripts installed in the virtual environment are accessible.
+
+To create a new virtual environment, you can use the `venv` module: `python -m venv .venv`
+
+Then, to activate the new virtual environment:
+
+PowerShell: `.venv\Scripts\Activate.ps1`
+Bash: `source .venv/bin/activate`
```

Comment:
I think it would also be worth adding the command for cmd,  `.venv\Scripts\activate.bat`